cli/tests/data/help/node.txt

```diff
@@ -6,6 +6,7 @@ Usage:
     dcos node --info
     dcos node [--json]
     dcos node log [--follow --lines=N --leader --master --mesos-id=<mesos-id> --slave=<slave-id>]
+                  [--component=<component-name> --filter=<filters>...]
```

Comment:
How will the user know which components exist? Should we add a `dcos node log list-components`?